packages/@aws-cdk/aws-kinesisfirehose-destinations/test/integ.s3-all-properties.ts

```diff
@@ -32,8 +32,8 @@ new firehose.DeliveryStream(stack, 'Delivery Stream', {
     errorOutputPrefix: 'errorPrefix',
     bufferingInterval: cdk.Duration.seconds(60),
     bufferingSize: cdk.Size.mebibytes(1),
+    encryptionKey: key,
   })],
-  encryptionKey: key,
```

Comment:
Note: this encryptionKey was never supposed to be here. I will be adding it to the DeliveryStream integ tests, since it is available as a property on the DeliveryStream, regardless of the destination in this PR: https://github.com/aws/aws-cdk/pull/15547/files